flink-runtime/src/test/java/org/apache/flink/runtime/io/network/buffer/LocalBufferPoolDestroyTest.java

```diff
@@ -20,30 +20,24 @@
 
 import org.apache.flink.runtime.execution.CancelTaskException;
 
-import org.junit.Test;
+import org.junit.jupiter.api.Test;
 
-import java.io.IOException;
 import java.util.concurrent.atomic.AtomicReference;
 
-import static org.junit.Assert.assertNotNull;
-import static org.junit.Assert.assertNull;
-import static org.junit.Assert.assertTrue;
-import static org.junit.Assert.fail;
+import static org.assertj.core.api.Assertions.assertThat;
+import static org.assertj.core.api.Assertions.assertThatThrownBy;
 
 /** Tests for the destruction of a {@link LocalBufferPool}. */
 public class LocalBufferPoolDestroyTest {
```

Comment:
```suggestion
class LocalBufferPoolDestroyTest {
```

Reply:
The static method `isInBlockingBufferRequest` is referenced by different package class `org.apache.flink.runtime.io.network.partition.ResultPartitionTest` and `org.apache.flink.runtime.taskmanager.TaskCancelAsyncProducerConsumerITCase`.

Reply:
Thanks, missed this before